apps/extension/webpack.config.ts

```diff
@@ -147,10 +190,11 @@ const config: webpack.Configuration = {
       filename: 'offscreen.html',
       chunks: ['offscreen-handler'],
     }),
+    // watch tarballs for changes
+    WEBPACK_WATCH && new WatchExternalFilesPlugin({ files: localPackages }),
+    WEBPACK_WATCH && PnpmInstallPlugin,
```

Comment:
was aware of built-in hot module reloading for webpack config (https://webpack.js.org/configuration/watch/), but wasn't aware of the watch options (https://webpack.js.org/configuration/watch/).